pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -468,11 +508,16 @@ func newMigration(name string, vmi string, phase v1.VirtualMachineInstanceMigrat
 	return migration
 }
 
-func newEvictionStrategy() *v1.EvictionStrategy {
+func newEvictionStrategyLiveMigrate() *v1.EvictionStrategy {
 	strategy := v1.EvictionStrategyLiveMigrate
 	return &strategy
 }
 
+func newEvictionStrategyLiveMigrateNone() *v1.EvictionStrategy {
```

Comment:
maybe `newEvictionStrategyNone` will be better

Reply:
yup it does look better, I'll change it, thanks!